src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/sealedclaim/SealedClaimResponseFormGeneratorForSpec.java

```diff
@@ -141,6 +140,12 @@ public SealedClaimResponseFormForSpec getTemplateData(CaseData caseData, String
         return builder.build();
     }
 
+    private void addRepaymentPlanDetails(SealedClaimResponseFormForSpec.SealedClaimResponseFormForSpecBuilder builder, CaseData caseData) {
+        if (featureToggleService.isLipVLipEnabled() && caseData.getRespondent2() == null) {
```

Comment:
If its multiparty scenario but respondent 1 has responded then do we want this code to run?

Reply:
no probably not. you are right. i am going to add the single one v one check instead